ml_data_analysis_code_NB.Rmd

```diff
@@ -30,7 +33,7 @@ library(pander) # to make some clean tables
 
 ``` {r include = TRUE}
 
-source(simulate-data.R)
+source(here(simulate-data.R))
```

Comment:
I'm not sure if this will work. If this will work, feel free to ignore. Generally, I've always used here() to grab the absolute path. So the use would be more like here() + "relative path". In code, this would be `paste0(here(), "relative path")`.